code/clue.js

```diff
@@ -10,50 +10,223 @@ const mrGreen = {
+  lastName: 'White',
+  color: 'white',
+  description: '',
+  age: 65,
+  image: 'assets/white.png',
+  occupation: 'Housekeeper'
+}
+
+const professorPlum = {
+  firstName: 'Victor',
+  lastName: 'Plum',
+  color: 'purple',
+  description: '',
+  age: 36,
+  image: 'assets/plum.png',
+  occupation: 'Professor'
+}
+
+const missScarlet = {
+  firstName: 'Cassandra',
+  lastName: 'Scarlet',
+  color: 'red',
+  description: '',
+  age: 25,
+  image: 'assets/scarlet.png',
+  occupation: 'Actress'
+}
+
+const mrsPeacock = {
+  firstName: 'Elizabeth',
+  lastName: 'Peacock',
+  color: 'blue',
+  description: '',
+  age: 32,
+  image: 'assets/peacock.png',
+  occupation: 'Socialite'
+}
+
+const colonelMustard = {
+  firstName: 'Jack',
+  lastName: 'Mustard',
+  color: 'yellow',
+  description: '',
+  age: 55,
+  image: 'assets/mustard.png',
+  occupation: 'Colonel'
+}
+
+
+// OBJECTS FOR ALL THE WEAPONS
 
 const rope = {
   name: 'rope',
   weight: 10
 }
 
-// THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
+const knife = {
+  name: 'knife',
+  weight: 7
+}
+
+const candlestick = {
+  name: 'candlestick',
+  weight: 4
+}
+
+const dumbbell = {
+  name: 'dumbbell',
+  weight: 20
+}
+
+const poison = {
+  name: 'poison',
+  weight: 1
+}
+
+const axe = {
+  name: 'axe',
+  weight: 15
+}
+
+const bat = {
+  name: 'bat',
+  weight: 9
+}
+
+const trophy = {
+  name: 'trophy',
+  weight: 9
+}
+
+const pistol = {
+  name: 'pistol',
+  weight: 6
+}
+
 
-// NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
+// ALL SUSPECTS, WEAPONS AND ROOMS GROUPED IN ARRAYS
 
 const suspects = [
   mrGreen,
-  mrsWhite
-  // ...  and the rest
+  mrsWhite,
+  professorPlum,
+  missScarlet,
+  mrsPeacock,
+  colonelMustard,
+]
+
+const weapons = [
+  rope,
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol
 ]
 
-const weapons = []
+// Rooms does not need an object because the rooms have only one property
+
+const rooms = [
+  'dining room',
+  'conservatory',
+  'kitchen',
+  'study',
+  'library',
+  'billiard room',
+  'lounge',
+  'ball room',
+  'hall',
+  'spa',
+  'living room',
+  'observatory',
+  'theater',
+  'guest house',
+  'patio'
+]
+
+// Function for reload of page, to play again
+const reload = () => {
+  location.reload();
+}
 
-const rooms = []
+document.getElementById('playAgain').onclick = reload
 
-// THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
-// YOU DON'T NEED TO CHANGE THIS, JUST TRY TO UNDERSTAND IT. AND HOW TO USE IT.
+
+// THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT IS PASSED IN THE FUNCTION
 const randomSelector = array => {
   return array[Math.floor(Math.random() * array.length)]
 }
 
-// CREATE AN OBJECT THAT KEEPS THE MYSTERY.
+
+// OBJECT THAT KEEPS THE MYSTERY.
 // With a killer, a weapon and a room.
 // The values will be set later.
+const mystery = {
+  killer: 'unknown',
+  weapon: '',
+  room: ''
+}
```

Comment:
It would be good to be consistent here.  Assign all of these values the same thing.
Good options would be `null` or `undefined`

```javascript
const mystery = {
  killer: undefined,
  weapon: undefined,
  room: undefined
}
```

This would allow you to make a smarter reveal function on line 227.  To make sure you reveal only after the user has clicked all the cards 👍